deploy-deps.sh

```diff
@@ -3,12 +3,48 @@
 script_path="$(dirname -- "${BASH_SOURCE[0]}")"
 
 main() {
+    echo "Checking requirements" >&2
+    check_req
     echo "Deploying Konflux Dependencies" >&2
     deploy
     echo "Waiting for the dependencies to be ready" >&2
     "${script_path}/wait-for-all.sh"
 }
 
+check_req(){
+    # declare the requirements
+    requirements=(kubectl openssl)
+    uninstalled_requirements=()
+
+    # check if requirements are installed
+    for i in ${requirements[@]}
+    do
+        if ! command -v $i &> /dev/null; then
+                uninstalled_requirements+=("$i")
+        else
+                echo -e "$i is installed"
+    fi
+    done
+
+    # check if podman or docker are installed 
+    if ! command -v podman &> /dev/null && ! command -v docker &> /dev/null; then
```

Comment:
why do we need to check they are installed?

Reply:
We checked it and when we are running the kind cluster creation we already need podman or docker, so the `deploy-deps.sh` should not check for it.
I will remove it

Reply:
Removed